api/base/settings/defaults.py

```diff
@@ -54,6 +54,7 @@
 
 REST_FRAMEWORK = {
     'PAGE_SIZE': 10,
+    'DEFAULT_BULK_LIMIT': 10,
```

Comment:
This is a bit misleading because this is not actually a DRF setting. It might make sense to separate this.